app/experimenter/experiments/models.py

```diff
@@ -44,6 +44,9 @@ class Experiment(ExperimentConstants, models.Model):
     owner = models.ForeignKey(
         get_user_model(), blank=True, null=True, on_delete=models.CASCADE
```

Comment:
Add a `related_name=owned_experiments` here and update references to it in the templates